packages/api-electron/package.json

```diff
@@ -2,14 +2,15 @@
   "name": "containerjs-api-electron",
   "version": "0.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "build/lib/preload.js",
```

Comment:
Good point, and I haven't tested it, but I think that the path to the entry point of the application is [explicitly provided](https://github.com/symphonyoss/ContainerJS/blob/3d3a1c7cb9f130d7d6c8b986764e828ace9c9454/packages/api-electron/bin/ssf-electron#L8) in the CLI, so the `main` entry point will not be used:

```js
var mainFile = path.join(__dirname, '../main.js');

// add the location of the main JS file to the command line args
var args = [mainFile, '--enable-sandbox'].concat(process.argv);

var child = proc.spawn(electron, args, {stdio: 'inherit'})
```